src/core/createEventManager.js

```diff
@@ -10,15 +10,27 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
-import { clone, noop } from "../utils";
+import { clone } from "../utils";
 
 export default ({ createEvent, optIn, lifecycle, network, config }) => {
   const { imsOrgId } = config;
 
   return {
     createEvent,
+    /**
+     * Sends an event. This includes running the event and payload through
+     * the appropriate lifecycle hooks, sending the request to the server,
+     * and handling the response.
+     * @param {Object} event This will be JSON stringified and used inside
+     * the request payload.
+     * @param {Object} [options]
+     * @param {boolean} [options.isViewStart=false] Whether the event is a
+     * result of the start of a view. This will be passed to components
+     * so they can take appropriate action.
+     * @returns {*}
+     */
     sendEvent(event, options = {}) {
-      const { isViewStart = false, applyUserProvidedData = noop } = options;
+      const { isViewStart = false } = options;
```

Comment:
It seems odd that we pass `isViewStart` around separately from `event`, but we manage `isDocumentUnloading` and `expectsResponse` directly on the event object itself. Does anyone else think we should move `isViewStart` into the event itself?

Reply:
I agree, good catch.

Reply:
+1

Reply:
@Aaronius we have `isViewStart` and `isDocumentUnloading()` and then `expectsResponse()` should we rename it to `isExpectingResponse()`? Thoughts?

Reply:
Yeah, probably. I'll make a separate issue to track this.

Reply:
For reference: https://jira.corp.adobe.com/browse/CORE-37113